src/app/showcases/[slug]/pdf/page.tsx

```diff
@@ -0,0 +1,17 @@
+'use client'
+
+import { useParams } from 'next/navigation';
```

Comment:
Don't forget to add these page into robots.txt. They should not be indexed by Google.